docs/local_install_skaffold.md

```diff
@@ -0,0 +1,190 @@
+# Local install of Substra using kubernetes and skaffold
+
+This guide will help you run the Substra platform on your machine with a 2 organisations setup.
```

Comment:
We could add that this is the development mode.

Reply:
👍